browser-test/bin/wait_for_server_start_and_run_tests.sh

```diff
@@ -25,6 +25,8 @@ for arg; do
   set -- "$@" "$arg"
 done
 
+yarn install
```

Comment:
Mind adding a comment as to why this is needed?

Can it go inside the $debug `if` below?

Reply:
I moved it from "build image" phase to "run test" phase. The rationale is that having it during "build image" doesn't really optimize for anything AFAIK. And having it in "run test" phase means that local changes to dependencies will be taken into effect. Like if someone adds a new dependency for TS browser tests - by running `bin/run-browser-tests` it will fetch dependencies and use them. With the old configuration changes to dependencies require rebuilding image locally. 

Also it matches better `server` where you can change dependencies in `build.sbt` and it will take effect immediately when running `run-dev` without need to rebuild image.